### ISSUE TITLE
[Features] Add support for Kitti semantic Segmentation dataset

### ISSUE THREAD

**Issue by AkideLiu** [open-mmlab/mmsegmentation#1599]
# Describe the feature

Kitti semantic Segmentation dataset is a lightweight dataset for semantic segmentation which shares the same label policy as cityscapes. It's an excellent starting point for segmentation and employs the weights pre-trained on cityscapes to perform transfer-learning, do you consider to support this dataset. Happy to create a PR. 

http://www.cvlibs.net/datasets/kitti/eval_semseg.php?benchmark=semantics2015

**Motivation**
A clear and concise description of the motivation of the feature.
Ex1. It is inconvenient when [....].
Ex2. There is a recent paper [....], which is very helpful for [....].

**Related resources**
If there is an official code release or third-party implementations, please also provide the information here, which would be very helpful.

**Additional context**
If you would like to implement the feature and create a PR, please leave a comment here and that would be much appreciated.

**Comment by MeowZheng** [comment]
Thanks in advance for your contribution to mmseg, and looking forward to your nice PR.

**Comment by AkideLiu** [comment]
> Thanks in advance for your contribution to mmseg, and looking forward to your nice PR.

Thanks for your quick reply, I have the following questions looking forward to your reply. 

 As this dataset has 200 training images with the fine label, however, the test set is not provided, do I need to implement the train_test_split for this dataset. In my current using scenario, the training set has been split from the original training set with 150 images and 50 images for both testing and validation datasets.  

if the train_test_split is required, could you please provide a guideline for the split ratio and reference code for insight?
